app/views/shared/_rollbar.html.erb

```diff
@@ -29,19 +29,43 @@
     }
   }
 
+  // Only report if env exists.
+  function filterTelemetry(e) {
```

Comment:
@kevinrobinson Why are we only reporting telemetry is env exists?

Reply:
Ah yeah good question.  I had thought this was similar to what we do for normal events but it shouldn't be and we can remove this.

For normal Rollbar events, the `transform` right above checks to see if env is set.  This happens because we load Rollbar independently from the rest of our app's code so that they're isolated from each other.  As a result, when Rollbar loads and then catches some kind of error, we don't know if the rest of the app's JS loaded or if something went wrong that prevented it.  If we can we get the env stuff and add it but if not we still report.  None of that really applies to telemetry (they will also be piped through the `transform` fn).

Reply:
👍